services/LARService.js

```diff
@@ -18,8 +18,17 @@ var compareYearTotals = function(newLoans, oldLoans, percentage) {
     return result;
 };
 
-var comparePercentages = function (newPercentage, oldPercentage, threshold) {
-    var diff = newPercentage - oldPercentage;
+var checkMaximumPercentageChange = function (newPercentage, oldPercentage, threshold) {
```

Comment:
`checkMaximumPercentageChange()` and `comparePercentages()` are confusing.. they check the same thing but return the opposite values. I bet you could use a single method.